DESCRIPTION

```diff
@@ -1,11 +1,20 @@
 Package: webapp
 Type: Package
-Title: What the Package Does (Title Case)
-Version: 0.1.0
-Author: Who wrote it
-Maintainer: The package maintainer <yourself@somewhere.net>
-Description: More about what it does (maybe more than one line)
-    Use four spaces when indenting paragraphs within the Description.
-License: What license is it under?
+Title: Display Sales Data
+Version: 0.0.0.9003
```

Comment:
Why that version? Let's start with sth 1ish.

Reply:
done